.gitignore

```diff
@@ -8,3 +9,9 @@ yarn-error.log
 # coverage
 coverage
 .nyc_output
+
+# output
+lib
```

Comment:
How come we are ignoring the output now?

Reply:
The output (`lib` folder) should be used for publishing. I would prefer to ignore it because then we have to make sure to not commit the stale output. It is basically the compiled result.